src/suspendplugin.c

```diff
@@ -253,23 +255,11 @@ suspend_handle_display_on_off_iface(
         sizeof(struct testmode_cmd_suspend),
         (void*)&susp_cmd);
 
-    err = 1;
-    nl_cb_err(cb, NL_CB_CUSTOM, handle_nl_command_error, &err);
-    nl_cb_set(cb, NL_CB_VALID, NL_CB_CUSTOM, handle_nl_command_valid, &err);
-    nl_cb_set(cb, NL_CB_FINISH, NL_CB_CUSTOM, handle_nl_command_finished, &err);
-    nl_cb_set(cb, NL_CB_ACK, NL_CB_CUSTOM, handle_nl_command_ack, &err);
-
-    if (nl_send_auto_complete(nl_socket, msg) < 0) {
+    if (nl_send_auto(nl_socket, msg) < 0) {
```

Comment:
Not sure whether it's important or not but I don't like this:
```
==3497== Syscall param sendmsg(msg.msg_iov[0]) points to uninitialised byte(s)
==3497==    at 0x4DC2F90: sendmsg (syscall-template.S:81)
==3497==    by 0x5620ECD: nl_sendmsg (nl.c:342)
==3497==    by 0x562100D: nl_send_iovec (nl.c:407)
==3497==    by 0x5621059: nl_send (nl.c:459)
==3497==    by 0x401C813: suspend_handle_display_on_off_iface (suspendplugin.c:258)
==3497==    by 0x401CAEF: suspend_handle_display_off (suspendplugin.c:298)
==3497==    by 0x401D2D7: suspend_plugin_init (suspendplugin.c:483)
==3497==    by 0x55607: __connman_plugin_init (plugin.c:220)
==3497==    by 0x53C07: main (main.c:870)
==3497==  Address 0x54952e1 is 41 bytes inside a block of size 4,096 alloc'd
==3497==    at 0x4841BF0: calloc (vg_replace_malloc.c:711)
==3497==    by 0x561F999: __nlmsg_alloc (msg.c:275)
==3497==    by 0x401C76B: suspend_handle_display_on_off_iface (suspendplugin.c:242)
==3497==    by 0x401CAEF: suspend_handle_display_off (suspendplugin.c:298)
==3497==    by 0x401D2D7: suspend_plugin_init (suspendplugin.c:483)
==3497==    by 0x55607: __connman_plugin_init (plugin.c:220)
==3497==    by 0x53C07: main (main.c:870)
```

Reply:
@monich this one is not obvious to me, will check more

Reply:
@monich one struct member was uint8_t and i forgot to initialize the padding...